bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterName.java

```diff
@@ -45,9 +47,28 @@ public String toString() {
   }
 
   /**
-   * @return The id of the instance that contains this cluster.
+   * @return The id of the instance that contains this cluster. It's the second group in the Cluster
```

Comment:
Hm? the following has two groups, no?

"projects/[^/]+/instances/([^/]+)/clusters/([^/]+)"
!=
"projects/(.+)/instances/(.+)/clusters/(.+)"